drools-osgi/drools-karaf-itests-kjar/src/main/java/org/dummypackage/DummyClass.java

```diff
@@ -0,0 +1,10 @@
+package org.dummypackage;
+
+/**
+ * Created by jpetrlik on 8/25/16.
+ */
```

Comment:
Nit-picking, but we should not be using author tags (see https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/blob/master/README.md#faq for explanation).

The class is also missing licence header.